components/layout/header.vue

```diff
@@ -9,7 +9,8 @@
             <BrandLogo class="w-24 h-24" />
           </NuxtLink>
         </div>
-        <div class="-mr-2 -my-2 md:hidden">
+        <div class="font-medium text-gray-800">Hotline 24/7</div>
+        <div class="mr-2 my-2 md:hidden">
```

Comment:
But I really like how it works and look:)

Reply:
Thank you